rst/cartridge_admin.rst

```diff
@@ -47,6 +47,104 @@ You can do this via the Web interface which is available at
+and *instance uuids* that **differ** from original cluster.
+
+Several notes:
+
+- You can bootstrap specific replicasets from a cluster
+  (for example, data nodes only) instead of the whole cluster.
+
+- Don't load data in the target cluster while bootstrapping.
+  If you need to hot switch between original and target cluster, stop data loading
+  in original cluster unless bootstrapping is completed.
+
+- Check logs and ``box.info.replication`` on target cluster after bootstrapping.
+  If something went wrong, try again.
+
+Example of valid data in ``edit_topology`` request:
+
+Original cluster topology:
+
+..  code-block:: javascript
+
+    {
+        "replicasets": [
+            {
+                "alias": "router-original",
+                "uuid": "aaaaaaaa-aaaa-0000-0000-000000000000",
+                "join_servers": [
+                    {
+                        "uri": "localhost:3301",
+                        "uuid": "aaaaaaaa-aaaa-0000-0000-000000000001"
+                    }
+                ],
+                "roles": ["vshard-router", "failover-coordinator"]
+            },
+            {
+                "alias": "storage-original",
+                "uuid": "bbbbbbbb-0000-0000-0000-000000000000",
+                "weight": 1,
+                "join_servers": [
+                    {
+                        "uri": "localhost:3302",
+                        "uuid": "bbbbbbbb-bbbb-0000-0000-000000000001"
+                    }
+                ],
+                "roles": ["vshard-storage"]
+            }
+        ]
+    }
+
+Target cluster topology:
+
+..  code-block:: javascript
+
+    {
+        "replicasets": [
+            {
+                "alias": "router-original",
+                "uuid": "cccccccc-cccc-0000-0000-000000000000", // <- this is dataless router,
+                // it's not necessary to bootstrap it from original cluster
+                "join_servers": [
+                    {
+                        "uri": "localhost:13301", // <- different uri
+                        "uuid": "cccccccc-cccc-0000-0000-000000000001"
+                    }
+                ],
+                "roles": ["vshard-router", "failover-coordinator"]
+            },
+            {
+                "alias": "storage-original",
+                "uuid": "bbbbbbbb-0000-0000-0000-000000000000", // replicaset_uuid is the same as in the original cluster
+                // that allows us bootstrap target cluster from original cluster
+                "weight": 1,
+                "join_servers": [
+                    {
+                        "uri": "localhost:13302", // <- different uri
+                        "uuid": "bbbbbbbb-bbbb-0000-0000-000000000002" // <- different instance_uuid
+                    }
+                ],
+                "roles": ["vshard-storage"]
+            }
+        ]
+    }
+
+
+-------------------------------------------------------------------------------
+Setup cluster
```

Comment:
For consistency, this should be either **Setting up cluster** (most of other headings are in the gerund form "Doing something") or **Cluster setup** (simple noun form).